htdocs/takepos/ajax/ajax.php

```diff
@@ -120,4 +120,13 @@
         $object->fetch($id);
         $ret = $printer->sendToPrinter($object, $conf->global->{'TAKEPOS_TEMPLATE_TO_USE_FOR_INVOICES'.$term}, $conf->global->{'TAKEPOS_PRINTER_TO_USE'.$term});
     }
+} elseif ($action == 'getInvoice') {
```

Comment:
We often forgot it but we should add a security test on permission on ajax component too.
Can you add a test on   $user->right->facture->lire ?